sinks/kafka/kafka_test.go

```diff
@@ -187,37 +188,41 @@ func TestSpanInstantiateFailure(t *testing.T) {
 	stats, _ := statsd.NewBuffered("localhost:1235", 1024)
 
 	// Busted duration
-	_, err := NewKafkaSpanSink(logger, "testing", "veneur_spans", "hash", "all", 1, 2, 3, "farts", "", stats)
+	_, err := NewKafkaSpanSink(logger, "testing", "veneur_spans", "hash", "all", 1, 2, 3, "farts", "", "", 1.0, stats)
```

Comment:
Can you add some assertions that verify that the constructor arguments set the correct fields in the resulting struct? Too often I've found bugs where the "write your own constructor" pattern results in us failing to set an argument into a field and not catching it. 😄

Reply:
👍  good call.